app/Http/Controllers/Store/BundleController.php

```diff
@@ -126,17 +125,40 @@ public function addVouchersToCurrentBundle(StoreAppendBundleRequest $request, Re
      */
     public function update(StoreUpdateBundleRequest $request, Registration $registration)
     {
-        // find our bundle.
+        $inputs = $request->only([
+            "collected_at",
+            "collected_by",
+            "collected_on"
+        ]);
+
+        // refactored to NOT alter vouchers IF we don't mention them in form input.
+        if ($request->exists('vouchers')) {
+            $inputs['vouchers'] = $request->input('vouchers');
```

Comment:
What is the case where no vouchers in request - but don't want them cleared? When updating the status only? Maybe alter comment to clarify `IF we don't mention them in form input because we are updating status of existing bundle vouchers`

Reply:
Yep, will change is clearer